pkg/config/setup/config.go

```diff
@@ -1135,6 +1136,7 @@ func serializer(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("serializer_max_series_payload_size", 512000)
 	config.BindEnvAndSetDefault("serializer_max_series_uncompressed_payload_size", 5242880)
 	config.BindEnvAndSetDefault("serializer_compressor_kind", DefaultCompressorKind)
+	config.BindEnvAndSetDefault("serializer_compressor_level", zstd.DefaultCompression)
```

Comment:
I think we should name it differently since it applies to zstd only, WDTY? `serializer_zstd_compressor_level`?